modules/custom-post-types/testimonial.php

```diff
@@ -592,8 +593,12 @@ static function jetpack_testimonial_shortcode_html( $atts ) {
 					<div class="testimonial-entry <?php echo esc_attr( self::get_testimonial_class( $testimonial_index_number, $atts['columns'] ) ); ?>">
 						<?php
 						// The content
-						if ( false !== $atts['display_content'] ): ?>
-							<div class="testimonial-entry-content"><?php the_excerpt(); ?></div>
+						if ( false !== $atts['display_content'] ): 
+							if ( true == $atts['show_excerpt'] ) { ?>
+								<div class="testimonial-entry-content"><?php the_excerpt(); ?></div>
+							<?php } else { ?>
+										<div class="testimonial-entry-content"><?php the_content(); ?></div>
+								<?php }
```

Comment:
You'll either need to add a closing php tag here, or omit the opening one in the line below